ISISScientificComputing/autoreduce
# Refactor QueueProcessors to suit pylint

### Summary of work
* QueueProcessors -> queue_processors
* QueueProcessor -> queue_processor
* AutoreductionProcessor -> autoreduction_processor

### How to test your work
* Add this code to the develop environment and ensure that all `QueueProcessors` facing code still works
* Ensure pylint on travis no longer references QueueProcessor problems

Fixes #123
